check.py

```diff
@@ -175,6 +175,31 @@ def do_asm2wasm_test():
               fail_with_error('wasm interpreter error: ' + err) # failed to pretty-print
             fail_with_error('wasm interpreter error')
 
+        # verify debug info
+        if 'debugInfo' in asm:
+          jsmap = 'a.wasm.map'
+          cmd += ['--binarymap-file', jsmap,
```

Comment:
yes, those names sound good.

to make sure i understand, the current state is now that we emit just 2 files, the wasm binary (which has a section that has the URL) and the JSON source map file (which that URL should point to)?